backend/framework/qlf/dashboard/bokeh/qlf_plot.py

```diff
@@ -548,12 +574,12 @@ def plot_amp(dz, mapper, name=""):
         "text_baseline": "middle"}
 
     p.rect("x", "y", .98, .98, 0, source=source,
-           fill_color={'field': 'z', 'transform': mapper}, fill_alpha=0.9)
+           fill_color={'field': 'zdiff', 'transform': mapper}, fill_alpha=0.9)
 
-    p.text(x="x", y="y_offset2", text="ztext",
-           text_font_style="bold", text_font_size="1.1vw", **text_props)
     p.text(x="x", y="y_offset1", text="amp",
-            text_font_size="1vw", **text_props)
+            text_font_size="2vw", **text_props)
+    p.text(x="x", y="y_offset2", text="ztext",
```

Comment:
Shouldn't this be the difference between the value and the reference?

Reply:
No, following the desispec. The color scales are related to the difference.